stdlib/collections/__init__.pyi

```diff
@@ -301,3 +310,13 @@ class ChainMap(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def pop(self, key: _KT) -> _VT: ...
     @overload
     def pop(self, key: _KT, default: _VT | _T = ...) -> _VT | _T: ...
+    def copy(self: Self) -> Self: ...
+    @classmethod
+    @overload
+    def fromkeys(cls, iterable: Iterable[_T]) -> ChainMap[_T, Any | None]: ...
+    @classmethod
+    @overload
+    def fromkeys(cls, __iterable: Iterable[_T], __value: None) -> ChainMap[_T, Any | None]: ...
```

Comment:
Does this really not work?

```suggestion
    @classmethod
    @overload
    def fromkeys(cls, iterable: Iterable[_T], __value: None = ...) -> ChainMap[_T, Any | None]: ...
```

Reply:
You're right, thanks! Didn't realise it was possible to do this in the stubs 🙂 I assumed it was illegal in the stubs to have a pos-only parameter after a pos-or-keyword parameter, since it's illegal to do that at runtime!